doc/source/basics.rst

```diff
@@ -466,6 +466,12 @@ preserve the location of NA values:
 
    df.cumsum()
 
+Also, refer to the ``expanding()`` function which is similar to the ``cumsum`` function.
+
+.. ipython:: python
+
+   df.expanding()
```

Comment:
Should be ``.expanding().sum()``, same as #12673 (link via docstring, use section link, or add explicit explanation).